src/interactive/azext_interactive/azclishell/scenario_suggest.py

```diff
@@ -0,0 +1,86 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import re
+
+from prompt_toolkit.auto_suggest import AutoSuggest, Suggestion
+
+
+class ScenarioAutoSuggest(AutoSuggest):
+    """Auto Suggest used in Scenario execution mode"""
+    def __init__(self):
+        self.cur_sample = ''
+        self.cur_command = ''
+        self.param_value_map = {}
+
+    def update(self, sample: str):
+        """Change command sample that would be suggested to the user"""
+        self.cur_sample = sample.split('az ')[-1]
+        self.param_value_map = {}
+        self.cur_command = self.cur_sample.split('-')[0].strip()
```

Comment:
Please add an example to explain this logic

Reply:
Done